slides/src/content/introduction.md

```diff
@@ -37,4 +53,247 @@ FIXME
   - Supporting projects: CLI, Universal, Nativescript
 - A complete and opinionated solution
 - You don't need to be a Javascript expert
-- Can be integrated with Redux
\ No newline at end of file
+- Can be integrated with Redux
+
+
+---
+
+## Getting Set Up
+
+1. Install NVM (Node Version Manager) -- [nvm.sh](http://nvm.sh)
+1. Install the latest LTS version of node -- `nvm install --lts`
+1. Install Angular CLI -- `npm install --global @angular/cli`
+
+---
+
+## Creating a New Project
+
+`ng create [project-name]`
+
+This will:
+
+- Install Angular
+- Install node dependencies
+- Create scaffolding
+
+---
+
+## Project Structure
+
+- `package.json` -- project metadata
+- `node_modules` -- libraries and dependencies used by our application
+- `src` folder -- application code
+- `tsconfig.json` -- TypeScript configuration
+
+---
+
+## TypeScript
+
+- TypeScript is exactly like JavaScript, but with types
+- Has ES6 and ES7 features
+- Compiles to JavaScript before being run in the browser
+- No runtime support for types
+
+---
+
+## First Look at the Component
+
+```typescript
```

Comment:
You put filenames in italics before chunks of code below - can you do that here as well please?